fuseimg/datasets/knight.py

```diff
@@ -187,6 +200,7 @@ def static_pipeline(data_path: str, resize_to: Tuple, test: bool = False) -> Pip
                 # loading data
                 (OpLoadImage(data_path), dict(key_in="data.input.img_path", key_out="data.input.img", format="nib")),
                 # fixed image normalization
+                # (OpNormalizeAgainstSelf(), dict(key="data.input.img")),
```

Comment:
If you are not using it, please delete it and also the definition above,

Reply:
why? its an option for the user to try it